cmplibrary/src/androidTest/java/com/sourcepoint/cmplibrary/data/local/DataStorageCcpaImplTest.kt

```diff
@@ -17,21 +17,21 @@ class DataStorageCcpaImplTest {
     fun check_DataStorage_TcData_gets_stored() {
         // Context of the app under test.
         val appContext = InstrumentationRegistry.getInstrumentation().targetContext
-        val storage = DataStorageCcpa.create(appContext).apply { clearAll() }
+        val storage = DataStorageCcpa.create(appContext).apply { deleteCcpaConsent() }
         val map = TreeMap<String, String>()
     }
```

Comment:
I don't get what this test case is actually testing, it seems like we initializing `dataStorage`, deleting CCPA consent on a go, and then do nothing